backend/src/main/java/com/festago/artist/application/ArtistV1CommandService.java

```diff
@@ -0,0 +1,31 @@
+package com.festago.artist.application;
+
+import com.festago.admin.dto.ArtistV1CreateRequest;
+import com.festago.admin.dto.ArtistV1UpdateRequest;
+import com.festago.artist.domain.Artist;
+import com.festago.artist.repository.ArtistRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class ArtistV1CommandService {
+
+    private final ArtistRepository artistRepository;
+
+    public Long save(ArtistV1CreateRequest request) {
+        return artistRepository.save(new Artist(request.name(), request.profileImage()))
+                .getId();
+    }
```

Comment:
사소한 부분이긴 한데, 체이닝을 다음과 같이 바꿔주는건 어떤가요?

```suggestion
    public Long save(ArtistV1CreateRequest request) {
        Artist artist = artistRepository.save(new Artist(request.name(), request.profileImage()))
        return artist.getId();
    }
```